release/images.sh

```diff
@@ -37,18 +37,11 @@ checksum_and_sign() {
 
         _output+="${file} ${chksum_file} " # space is not a mistake
 
-        if [[ -n "${IMG_SIGN_CMD}" ]]; then
-            log "Signing (custom)" "${chksum_file}"
-            "${IMG_SIGN_CMD}" "${chksum_file}" "${chksum_file}.sig"
-        elif [[ -s "${IMG_SIGN_KEY}" ]]; then
-            log "Signing (openssl)" "${chksum_file}"
-            openssl dgst -sha1 -sign "${IMG_SIGN_KEY}" -out "${chksum_file}.sig" "${chksum_file}"
```

Comment:
We would still like to support a default signing method for images without the need to provide custom command. 
Any particular reason for removing this support?

Reply:
After speaking offline, I'll put this back in.  I originally thought we could abstract signing away into a function, since that case could also cover the default signing method.  But we want to provide a default signing method rather than leaving it entirely up to the user :)